lib/angular/mediator-ng.js

```diff
@@ -2,9 +2,11 @@
 
 var mediator = require('../mediator');
 
+/* eslint-disable */
 angular.module('wfm.core.mediator', ['ng'])
+/* eslint-enable */
```

Comment:
Were these added to avoid `angular variable not defined` ? The best practice would be to [add a global](http://eslint.org/docs/user-guide/configuring#specifying-globals) I think we can add `angular` to the globals object in the eslintrc instead of per file.